router/submit_pkg_handler.go

```diff
@@ -0,0 +1,40 @@
+package router
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/vulpemventures/nigiri-chopsticks/helpers"
+)
+
+// HandleSubmitPackageRequest handles the submitpackage request
+// it mocks the incoming /txs/package endpoint
+// TODO remove once esplora supports it
+func (r *Router) HandleSubmitPackageRequest(res http.ResponseWriter, req *http.Request) {
+	res.Header().Set("Access-Control-Allow-Origin", "*")
+
+	decoder := json.NewDecoder(req.Body)
+	var txs []string
+	err := decoder.Decode(&txs)
+	if err != nil {
+		http.Error(res, "Malformed Request: missing txs", http.StatusBadRequest)
+		return
+	}
```

Comment:
please use `parseRequestBody` like done in the other handlers to keep consistency